services/notifications/src/app/features/notifications/models/notification.model.ts

```diff
@@ -0,0 +1,44 @@
+import { Entity, Column, CreateDateColumn, UpdateDateColumn, PrimaryGeneratedColumn } from "typeorm";
+
+interface NotificationModelProps {
+  chanel: string;
+  message: string;
+}
+
+@Entity({
+  name: "Notification",
+})
+export class NotificationModel {
+  public static create(data: Partial<NotificationModelProps>): NotificationModel {
+    const entity = new NotificationModel();
+    Object.assign(entity, data);
+
+    return entity;
+  }
+
+  public static createMany(channels: string[], message: string): NotificationModel[] {
+    const entities = channels.map((chanel) => {
+      const entity = new NotificationModel();
+      Object.assign(entity, { chanel, message });
+
+      return entity;
+    });
+
+    return entities;
+  }
+
+  @PrimaryGeneratedColumn("uuid")
+  id: string;
+
+  @Column({ nullable: false })
+  chanel: string;
```

Comment:
Create an index for the values ​​in this column.
`channel` is a string: userId (from token),` default-all`, `default-authorized`, or default-unauthorized`.

Reply:
I have added index 👍